aten/src/ATen/native/cuda/TensorTransformations.cu

```diff
@@ -69,7 +69,7 @@ void flip_cuda_kernel(scalar_t* in_tensor, scalar_t* out_tensor, int64_t N, int6
 Tensor flip_cuda(const Tensor& self, IntList dims) {
   auto in_tensor = self;
   const int64_t flip_dims_size = dims.size(), total_dims = in_tensor.dim(), N = in_tensor.numel();
-  flip_check_errors(total_dims, flip_dims_size, dims);
+  dim_list_to_bitset(dims, total_dims);
```

Comment:
yeah, you're right. The reason I use `dim_list_to_bitset` is to avoid duplicate dims: `dims = [0, 0]`, and I really want this check. Please correct me if you think it is not needed. Maybe I should introduce another one similar to `dim_list_to_bitset` without returning bitset since I might not need it.